provider_shopper/lib/models/cart.dart

```diff
@@ -16,9 +16,9 @@ class CartModel extends ChangeNotifier {
   CatalogModel get catalog => _catalog;
 
   set catalog(CatalogModel newCatalog) {
-    assert(newCatalog != null);
-    assert(_itemIds.every((id) => newCatalog.getById(id) != null),
-        'The catalog $newCatalog does not have one of $_itemIds in it.');
+    // // assert(newCatalog != null);
+    // assert(_itemIds.every((id) => newCatalog.getById(id) != null),
+    // 'The catalog $newCatalog does not have one of $_itemIds in it.');
```

Comment:
Either delete or update. Don't leave commented code in the repo.

Reply:
Yes, I will keep in mind this in my upcoming patches. Probably, for this case these lines of code will be required again as we are making `CatalogModel` as `nullable`.

Reply:
Ah, ok, I didn't realize this is still a draft PR. You can mark it as such on GitHub.